master/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -5,52 +5,46 @@ canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/i
 no code modifications to either. Canal is simply a deployment pattern
 for installing and configuring the projects to work together seamlessly as a
 single network solution from the point of view of the user and orchestration
 system.
 
 
-## Installation
+## Before you begin 
 
-When deploying your Kubernetes cluster, please make sure it meets the
-[requirements](#requirements--limitations) at the bottom of this page.
-An easy way to create a cluster which meets these requirements is by following
-[the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+- Ensure that your cluster meets the [requirements](../../../reqs/system).
+  An easy way to create a cluster which meets these requirements is by 
+  following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
 
-> **Note:** If you are upgrading from the Kubernetes
-[1.6](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#for-kubernetes-16)
-or [1.5](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#kubernetes-15)
-manifests from the (deprecated) Canal repo to the manifests here it is
-neccessary to [migrate your Calico configuration
-data](https://github.com/projectcalico/calico/blob/master/upgrade/v2.5/README.md)
-before upgrading. Otherwise, your cluster may lose connectivity after the
-upgrade.
+- If you are upgrading from the Kubernetes [1.6](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#for-kubernetes-16)
+  or [1.5](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#kubernetes-15)
+  manifests from the (deprecated) Canal repo to the manifests here it is
+  necessary to [migrate your {{site.prodname}} configuration data](https://github.com/projectcalico/calico/blob/master/upgrade/v2.5/README.md) 
+  before upgrading. Otherwise, your cluster may lose connectivity after 
+  the upgrade.
 
-### RBAC
+- If your Kubernetes cluster has RBAC enabled, you'll need to create RBAC roles 
+  to allow API access by Canal. Apply the following manifest to create these 
+  necessary RBAC roles and bindings.
 
-Before you install Canal, if your Kubernetes cluster has RBAC enabled, you'll
-need to create the following RBAC roles to allow API access by Canal.
-
-Apply the following manifest to create these necessary RBAC roles and bindings.
-
-```
-kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/canal/rbac.yaml
-```
-> **Note**: You can also [view the YAML in your browser.](rbac.yaml){:target="_blank"}.
-{: .alert .alert-info}
+   ```
+   kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/canal/rbac.yaml
+   ```
+   > **Note**: You can also [view the YAML in your browser.](rbac.yaml){:target="_blank"}.
+   {: .alert .alert-info}
 
 ### Canal with the Kubernetes API datastore
```

Comment:
Given we only have one option here, perhaps this heading should just be "Installing Canal"?